cmd/core-command/Dockerfile

```diff
@@ -18,11 +17,6 @@ RUN sed -e 's/dl-cdn[.]alpinelinux.org/nl.alpinelinux.org/g' -i~ /etc/apk/reposi
 
 RUN apk update && apk add make git
 
-COPY go.mod .
-#COPY go.sum .
-
-RUN go mod download
-
```

Comment:
I saw this as a redundant step. When the `go build` is eventually executed...it will take care of downloading any missing go mods. This should no affect anything for local dev builds